src/actions/actions.ts

```diff
@@ -19,7 +19,57 @@
 // THE SOFTWARE.
 
 import ActionTypes from 'constants/action-types';
-import {createAction} from 'redux-actions';
+import {createAction} from '@reduxjs/toolkit';
+
+import {ParsedConfig} from '../schemas';
+import {RGBColor} from 'reducers/types';
+import {Bounds} from 'reducers/map-state-updaters';
+import {MapInfo} from 'reducers/vis-state-updaters';
+import {UiState} from 'reducers/ui-state-updaters';
+
+/**
+ * Input dataest parsed to addDataToMap
```

Comment:
Fix typo. Not sure the rest of the comment makes sense but that is probably not your fault.
```suggestion
 * Input dataset (parsed to addDataToMap)
```